assignments/prototypes.js

```diff
@@ -16,12 +16,29 @@
   * destroy() // prototype method that returns: `${this.name} was removed from the game.`
 */
 
+const GameObject = function(attr){
+    this.createdAt = attr.createdAt;
+    this.name = attr.name;
+    this.dimensions = attr.dimensions;
+}
+GameObject.prototype.destroy = function(){
+  return `${this.name} was removed from the game.`
+}
+
 /*
   === CharacterStats ===
   * healthPoints
   * takeDamage() // prototype method -> returns the string '<object name> took damage.'
   * should inherit destroy() from GameObject's prototype
 */
+const CharacterStats = function(attr){
+   GameObject.call(this, attr)
+  this.healthPoints = attr.healthPoints;
+}
+CharacterStats.prototype.takeDamage = function(attr){
+  return `${this.name} took damage.`
+}
+CharacterStats.prototype = Object.create(GameObject.prototype);
```

Comment:
do this before you add the takeDamage method. i.e take this line 41 above line 38.